lib/iris/tests/test_std_names.py

```diff
@@ -18,15 +15,14 @@ class TestStandardNames(tests.IrisTest):
 
     def test_standard_names_table(self):
         # Check we have a dict
-        self.assertIsInstance(STD_NAMES, dict)
+        assert isinstance(STD_NAMES, dict)
 
         keyset = set(STD_NAMES)
 
         # Check for some known standard names
         valid_nameset = set(["air_density", "northward_wind", "wind_speed"])
```

Comment:
OOS, but can you think why they wouldn't have just initialised this as a set? or should it be added to the list of bad practice